src/Directory.build.props

```diff
@@ -39,7 +39,7 @@
   <ItemGroup>
     <PackageReference Include="stylecop.analyzers" Version="1.2.0-beta.205" PrivateAssets="all" />
```

Comment:
stylecop.analyzers 1.2.0-beta.312 has a bunch of improvements for C# 9 if you want to bump that as well.

Reply:
Sorry for the multiple iterations of comments btw.

Reply:
No worries, and thanks for reviewing this! Going to bump that version as well soon.